roles/oneagent/tasks/cleanup/cleanup-unix.yml

```
@@ -3,8 +3,10 @@
   ansible.builtin.file:
     path: "{{ oneagent_ca_cert_dest_path }}"
     state: absent
+  changed_when: false
 
 - name: Remove leftover installer
   ansible.builtin.file:
     path: "{{ oneagent_installer_path }}"
     state: absent
+  changed_when: false
```

I'd remove this to report correct state. It's a single file, so there should be no edge cases
```suggestion
  changed_when: false
```